src/main/java/com/zombiecastlerush/building/Room.java

```diff
@@ -38,6 +42,19 @@ public void setDescription(String description) {
         return connectedRooms;
     }
 
+    /**
+     *
+     * @return
+     */
+    @JsonGetter("connectedRooms")
+    public List<String> displayConnectedRooms(){
```

Comment:
Why do you need displayConnectRooms() in addition to getConnectedRooms()?

Reply:
getConnectedRooms() will lead to a infinite loop. that is why i added List<String> here. it is more compatible with Json format